server/src/router.ts

```diff
@@ -315,3 +342,34 @@ const spawnRates: { [timeSeconds: number]: number } = {
   80: 0.6,
   100: 0.7,
 };
+
+const removeImageAfterSecs = (secondsImageShown: Number) => {
+  backgroundState.isShowingImage = true;
+
+  if (backgroundState.imageInterval) {
+    clearInterval(backgroundState.imageInterval);
+    backgroundState.loopCounter = 0;
+  }
+
+  backgroundState.imageInterval = setInterval(() => {
+    backgroundState.loopCounter++;
+
+    if (backgroundState.loopCounter === secondsImageShown) {
+      removeImage();
+    }
+  }, 1000);
+};
+
+const removeImage = () => {
+  if (backgroundState.imageInterval) {
+    clearInterval(backgroundState.imageInterval);
+  }
+
+  backgroundState = {
+    loopCounter: 0,
+    isShowingImage: false,
+    currentBackground: "blank"
```

Comment:
consider handling the case of undefined, since it's a clearer terminology than "blank" which could also be "empty", "white", etc
for instance, currentBackground would start undefined, and any time in the client when currentBackground is undefined we know to set the background image to undefined